_data/simple-icons.json

```diff
@@ -5673,6 +5673,18 @@
             "hex": "FBBA00",
             "source": "https://de.wikipedia.org/wiki/Datei:JET.svg"
         },
+        {
+            "title": "Jet",
+            "slug": "jet_reporter",
+            "hex": "0087CD",
+            "source": "https://insightsoftware.com/jet/",
+            "aliases": {
+                "aka": [
+                    "Jet Reports",
+                    "Jet Hub"
```

Comment:
I found these names here: https://insightsoftware.com/downloads/.

These aliases are different pricing plans, right? I'd prefer to remove these aliases because they are not common product names.

Please correct me if I am wrong.

Reply:
Well, each of them are more different products. You've `Jet Reports` and `Jet Analytics` which are two different programs. The reason I added `Jet Reports` and `Jet Hub` is because they both have the same logo. But if it becomes to complex we can just use ´Jet´ as the complete package. Which is also the case.

Reply:
Let's keep `Jet` only. In fact, those aliases are only for search purposes. The `Jet` is enough to use.

Reply:
BTW, I didn't find `Jet Reporter` on their homepage. Shall we use `jetglobal` as its slug?

In the meantime, maybe we could change its title to `Jet Global`. Then we don't need to specify its slug manually anymore. What do you think?

Reply:
Yeah good idea! I'll change it tomorrow.